next/src/components/page-contents/GeneralPageContent.tsx

```diff
@@ -32,16 +34,10 @@ const GeneralPageContent = ({ page }: GeneralPageProps) => {
       </PageHeader>
 
       {/* Page - Common Sections */}
-      <div className="mb-8">
-        {page.attributes?.sections?.length ? (
-          <Sections sections={page.attributes.sections} />
-        ) : null}
-
-        <RelatedArticlesSection
-          page={page}
-          // The same as Section
-          className="pt-10 md:pt-18"
-        />
+      <div className="py-8">
+        <Sections sections={filteredSections} className="*:py-5 *:md:py-9" />
```

Comment:
I saw `md:` breakpoint used few times iin this PR. Is there a reason for switching for not using `lg:`?